app/models/effects/add_subject_to_collection.rb

```diff
@@ -14,6 +14,10 @@ def valid?
       collection_id.present?
     end
 
+    def self.config_fields
+      ["collection_id"]
```

Comment:
Config fields (here and in other models) should be frozen and probably constants